static/app/views/settings/organizationMembers/organizationMemberRow.tsx

```diff
@@ -216,7 +213,7 @@ export default class OrganizationMemberRow extends PureComponent<Props, State> {
                     : isPartnershipUser
                       ? t('You cannot make changes to this partner-provisioned user.')
                       : // only show this message if member can remove invites but invite was not sent by them
-                        isInvite && canRemoveInvites && !isInviteFromCurrentUser
+                        pending && canInvite && !isInviteFromCurrentUser
```

Comment:
does `pending` refer to whether the member has a `user` property?

Reply:
Sorry I updated the PR description, originally I was checking if `user` was null but then I realized there was a `pending` property which is true if the invite is still pending